Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [workspace]
 resolver = "2"
-members = [
+members = [ "benchmarks",
```

Comment:
Instead of creating a separate crate, can we not create this as a binary file in `miden-tx` crate?

Reply:
So the idea is to move the main.rs with `main()` to the `src` of the `miden-tx` to have a binary inside it, right?

Reply:
Yes, but we can also name it anything we want (doesn't have to be `main.rs`).

Also, it doesn't make it a binary crate. We'll have a library and a binary crate in `miden-tx`. This would work the same way as [miden-vm](https://github.com/0xPolygonMiden/miden-vm/blob/main/miden/Cargo.toml) crate works.